README.md

```diff
@@ -103,3 +103,35 @@ The syntax for the `sortby` attribute is:
     ]
 }
 ```
+
+## Sortables
+
+Additional endpoints that provide so called "Sortables" support clients that want to discover the list of resource
+properties with their types and constraints that may be used to sort resources.
+
+These Sortables endpoints return lists of properties (or aliases) that can be used in the `sortby` parameter.
+It returns a JSON Schema that defines the properties allowed in `sortby`.
+The precise definition of this can be found in the
+[OGC API - Features - Part 5: Schemas](https://portal.ogc.org/files/108199#rc_sortables).
+
+All Sortables endpoints SHALL be referenced with a link with the link relation type
+`http://www.opengis.net/def/rel/ogc/1.0/sortables`.
+
+| Sortables Endpoint                                          | Endpoint linking to the Sortables Endpoint | Conformance class                                                  | Applicable `sortby` endpoints           |
+| ----------------------------------------------------------- | ------------------------------------------ | ------------------------------------------------------------------ | --------------------------------------- |
+| `GET /sortables`                                            | `GET /`                                    | `https://api.stacspec.org/v1.0.0/item-search#sortables`            | `GET /search` and `POST /search`        |
```

Comment:
We may have here the same issue as per queryables. Are the sortables applicable to the collection search or items search. If item search, where are the collection search sortables?

Reply:
cc @vincentsarago

Reply:
I don't think there's an issue with it. /queryables and /sortables are for item search. For collection search you add a link to a customly named endpoint, see two rows below. It's the same way for queryables in collection search and documented in the specs as such. /collections/queryables (or sortables) may conflict with collection IDs as such we just follow the hypermedia approach.